libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -528,40 +528,50 @@ export class ListComponent extends CollectionBaseInput implements OnInit, AfterV
             });
     }
 
+
     /**@hidden
     *  used in tempate to get Selected items from a list
     * event:any to avoid code duplication**/
-    _onSelectionChanged(event: any): void {
-        if (event.target.checked) {
-            this._selectionModel.select(event.target.parentNode.parentNode.parentNode);
-        } else {
-            this._selectionModel.deselect(event.target.parentNode.parentNode.parentNode);
-        }
+    _onSelectionChanged(event: Event): void {
+        this._updateNavigation(event);
     }
 
     /** @hidden */
     /**  on Update navgiation styles for non navigated items
      * event:any to avoid code duplication
      * seprate PR for custom event**/
     @HostListener('click', ['$event'])
-    _updateNavigation(event: any): void {
+    _updateNavigation(event: Event): void {
+        let selectedItemId = '0';
+        const el = event.target as HTMLElement;
+        const parent = el.closest('.fd-list__item');
```

Comment:
I'm not sure if it's supported by IE11

Reply:
@sKudum

Reply:
Sure @droshev , as of now I am blocked with 1p task on high priority. will work on this PR once I get bandwidth. sorry for the delay(as its an enhancement PR(low priority) for list )

Reply:
@sKudum plz use `closestElement` instead of `closest`

Reply:
> @sKudum plz use `closestElement` instead of `closest`

Hi Yuri, I have tried using the closest element from but it is not returning expected values.

Reply:
when this [PR](https://github.com/SAP/fundamental-ngx/pull/3894) will be merged it will be working

Reply:
Updated with closestElement now